src/server/game/Spells/SpellMgr.cpp

```diff
@@ -2917,6 +2917,13 @@ void SpellMgr::LoadSpellInfoCorrections()
         spellInfo->AttributesEx3 |= SPELL_ATTR3_CANT_TRIGGER_PROC;
     });
 
+    // Revenge (Warrior) - DBC have SpellLevel & BaseLevel = 0 
```

Comment:
what does it fix?

Reply:
With only sql for proc with level <22 buff for revenge free appear in list of auras and get you free revenge, revenge is for level 22, that correct baselevel 0 in dbc.

Reply:
Like joschiwald said in https://github.com/TrinityCore/TrinityCore/pull/20394#issuecomment-340783194,
> add a comment why the dbc correction is needed to code too

before we can change the PR status to [Feedback-PatchFix].